CAM Platform/api-providers.md

```diff
@@ -11,41 +12,47 @@ Manage and perform provider actions.
 
 |  Resource  |  Description |
 |------------|--------------|
-| POST /services/providers | Creates a new provider account in Cloud Application Manager and gets the status of the provider. |
-| GET /services/providers | Gets the list of providers. |
-| GET /services/providers/{provider_id} | Fetches an existing provider. |
-| PUT /services/providers/{provider_id} | Updates an existing provider. |
-| DELETE /services/providers/{provider_id}| Deletes an existing provider.|
+| [POST /services/providers](#post-servicesproviders) | Creates a new provider account in Cloud Application Manager and gets the status of the provider. |
+| [GET /services/providers](#get-servicesproviders) | Gets the list of providers. |
+| [GET /services/providers/{provider_id}](#get-servicesprovidersprovider_id) | Fetches an existing provider. |
+| [PUT /services/providers/{provider_id}](#put-servicesprovidersprovider_id) | Updates an existing provider. |
+| [DELETE /services/providers/{provider_id}](#delete-servicesprovidersprovider_id) | Deletes an existing provider.|
 
 ### Provider Operations
 
 |  Resource  |  Description |
 |------------|--------------|
-| PUT /services/providers/{provider_id}/sync |Syncs an existing provider.|
-| GET /services/providers/{provider_id}/logs | Gets the logs of a provider. |
-| GET /services/providers/{provider_id}/unregisted-instances | Gets the unregistered instances of a provider. |
-| POST /services/providers/{provider_id}/images | Adds a new machine image to a provider. |
-| DELETE /services/providers/{provider_id}/images/{machine_image_id} | Updates an existing provider. |
+| [PUT /services/providers/{provider_id}/sync](#put-servicesprovidersprovider_idsync) |Syncs an existing provider.|
+| [GET /services/providers/{provider_id}/logs](#get-servicesprovidersprovider_idlogs) | Gets the logs of a provider. |
+| [GET /services/providers/{provider_id}/unregisted-instances](#get-servicesprovidersprovider_idunregisted-instances) | Gets the unregistered instances of a provider. |
+| [POST /services/providers/{provider_id}/images](#post-servicesprovidersprovider_idimages) | Adds a new machine image to a provider. |
+| [DELETE /services/providers/{provider_id}/images/{machine_image_id}](#delete-servicesprovidersprovider_idimagesmachine_image_id) | Updates an existing provider. |
 
 
-### POST /services/providers
+## POST /services/providers
 
 Creates a new provider account in Cloud Application Manager and gets the status of the provider.
 
-**Normal Code**
+### URL
 
-* 202 accepted
+#### Structure
 
-**Error Codes**
+    [POST] /services/providers
 
-**Request Headers**
+#### Example
 
+    POST https://cam.ctl.io/services/providers
+
+### Request
+
+#### Headers
 ```
 content-type:application/json
 Authorization: Bearer your_json_web_token
 ElasticBox-Release: 4.0
 ```
-**Request parameters for all providers**
+
+#### Request parameters for all providers
```

Comment:
Please edit the `type` parameter below to include the provider types in `resources.py` (except the Test Provider)

Reply:
Please also rewrite the `owner` to state `workspace ID` instead of `user`.